contracts/contracts/mocks/MintableERC20.sol

```diff
@@ -20,4 +22,15 @@ contract MintableERC20 is IMintableERC20, ERC20 {
         _mint(msg.sender, value);
         return true;
     }
+
+    /**
+     * @dev Function to burn the minted tokens
+     * @param account The address with the tokens to burn.
+     * @param value The amount of tokens to burn.
+     * @return A boolean that indicates if the operation was successful.
+     */
+    function burn(address account, uint256 value) public returns (bool) {
```

Comment:
We have MintableERC20.sol and BurnableERC20.sol, although the burn function has a different signature. We should try clean this up somehow, either using the existing burn function or add another with this signature.